frontend-react/package.json

```diff
@@ -123,6 +123,7 @@
         "eslint": "^7.26.0",
         "eslint-config-airbnb": "^18.2.1",
         "eslint-config-prettier": "^8.3.0",
+        "eslint-config-react-app": "^6.0.0",
```

Comment:
was this added by you? or just the result or running yarn?

Reply:
It was a result of running yarn, but I included it due to the new dependencies with the new fe dockerfile

Reply:
It was a result of running yarn, but I included it due to the new dependencies with the new fe dockerfile